src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -121,6 +121,24 @@ object WdlExpression {
           case "UnaryNegation" => for(e <- expression) yield e.unaryMinus.get
           case _ => throw new WdlExpressionException(s"Invalid operator: ${a.getName}")
         }
+      case a: Ast if a.getName == "ArrayLiteral" =>
+        val evaluatedElements = a.getAttribute("values").asInstanceOf[AstList].asScala.toVector map {x =>
+          evaluate(x, lookup, functions, interpolateStrings)
+        }
+        evaluatedElements.partition {_.isSuccess} match {
+          case (_, failures) if failures.nonEmpty =>
+            val message = failures.collect {case f: Failure[_] => f.exception.getMessage}.mkString("\n")
+            throw new WdlExpressionException(s"Could not evaluate expression:\n$message")
```

Comment:
Hmmmm yeah good question... I don't really have a good answer, I could change them too to `Failure` since that might make more sense?